utils/parsing.py

```diff
@@ -265,6 +265,8 @@ def brat_parse_to_bigbio_kb(brat_parse: Dict, entity_types: Iterable[str]) -> Di
     id_to_event_trigger = {}
     for ann in brat_parse["text_bound_annotations"]:
         if ann["type"] in entity_types:
+            if len(brat_parse["events"]) > 0:
```

Comment:
I am not sure if this is an update from the repo or you modified it. 
In the latter case, if you feel there is a bug in `parsing.py` I suggest you make a separate PR for that.

Reply:
I see. I will remove this line and copy the modified function into `seth_corpus.py`. It's the better way to avoid conflict with other files.